Contact form

hetrox8 opened the pull request:
This branch consist of

1. textarea 
2. name and email input

vickymarz reviewed (changes requested):
Hi @hetrox8  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project that has failed linter check. KIndly fix your linter errors and submit again 👍🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

hetrox8 commented:
sure


On Wed, May 10, 2023 at 12:29 AM Victor Ebube Okoroji <
***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @hetrox8 <https://github.com/hetrox8>
>
> While you made a great effort in this project, unfortunately, I cannot
> proceed to review your code.
> ❌ Invalid Code Review Request
>
> You have submitted a project that has failed linter check. KIndly fix your
> linter errors and submit again 👍🏽
>
> Your Code Review Request will be marked as invalid in your Dashboard, so
> please submit a new one once you are ready 🙏
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *Invalid Code Review Request does not count into the code reviews limit.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hetrox8/portfolio1/pull/8#pullrequestreview-1419483229>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ6AOJAIAJN2PZNYQM2TY7DXFKZJ3ANCNFSM6AAAAAAX326U7M>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

hetrox8 commented:
add user input fields
add user textarea 
add user button

wayungi commented on index.html:
```diff
@@ -261,17 +264,34 @@ <h3 class="lang">languages <a href="#"> <img src="./images/down-arrow.svg" class
+I’d love to help with it!
+        </p>
+
+        <form action="" method="post">
```
- [ ] Please use `flexbox` to lay out the form elements. This is a project requirement and flexbox simplifies implementing responsiveness 🛩️

wayungi commented on index.html:
```diff
@@ -261,17 +264,34 @@ <h3 class="lang">languages <a href="#"> <img src="./images/down-arrow.svg" class
+         <fieldset> 
+        <textarea name="message" placeholder="Enter your message here..."  cols="50" rows="10" maxlength="500" required></textarea>
+      </fieldset>
+       <button class="resume">
+         Get in touch
+       </button>
```
- [ ] Kindly implement the button `pressed` state to match the Figma design 👍🏾

wayungi reviewed (changes requested):
Hi @hetrox8 , 

![image](https://user-images.githubusercontent.com/99656687/234198503-c41d0396-d5b0-4466-8ed1-959de33796fa.png)


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] Your portfolio closely resembles the Figma 👏🏾 
- [ ] Your `readme` is well written 👏🏾 
- [ ] YOu followed the correct GitHub flow 👏🏾 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hetrox8 commented on index.html:
```diff
@@ -261,17 +264,34 @@ <h3 class="lang">languages <a href="#"> <img src="./images/down-arrow.svg" class
+        <form action="" method="post">
+          <fieldset> 
+          <input id="first-name" name="" type="text" required placeholder="lores ipsum" max-length="30" />
+         <input id="last-name" name="" type="email" required placeholder="loresipsum@gmail.com" />
```
ooh yea that  would Good

hetrox8 commented:
Added form action
use flexbox for laying out form elements
Changed the email Id to the required one
Add input validation for a maximum of 30 characters
Implement button effects ie. when pressing or hovered  around

AlphaNtihinduka reviewed (changes requested):
## Hello Team 🙋‍♂️🙋‍♂️
Good job working on implementing the contact form. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Implemented input type ✔️
- Your project is professional ✅


# STATUS: Required Changes :recycle: 

![close](https://user-images.githubusercontent.com/99394374/210771148-619a4265-33b2-4e68-8901-6bb52693d6de.gif)

## Required changes :recycle: 
- [ ] As the previous reviewer suggested, Kindly apply the changes and commit them to the Github repository. According to your communication, it seems you applied changes but you haven't committed them to the remote repository.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

hetrox8 commented:
I mistakenly forgot to push the changes i made . 
They include :
Added form action
use flexbox for laying out form elements
Changed the email Id to the required one
Add input validation for a maximum of 30 characters
Implement button effects ie. when pressing or hovered around

hetrox8 commented:
Done.

On Wed, May 10, 2023 at 11:22 AM NTIHINDUKA ALPHA ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> Hello Team 🙋‍♂️🙋‍♂️
>
> Good job working on implementing the contact form. You have done the
> design to do the expectation and incredibly well. 👏
>
> TO HIGHLIGHT:
>
>    - Implemented input type ✔️
>    - Your project is professional ✅
>
> STATUS: Required Changes ♻️
>
> [image: close]
> <https://user-images.githubusercontent.com/99394374/210771148-619a4265-33b2-4e68-8901-6bb52693d6de.gif>
> Required changes ♻️
>
>    - As the previous reviewer suggested, Kindly apply the changes and
>    commit them to the Github repository. According to your communication, it
>    seems you applied changes but you haven't committed them to the remote
>    repository.
>
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you to take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread or slack
> channel if something is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> HAPPY CODING AND GOOD LUCK 🤞 🍀
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hetrox8/portfolio1/pull/8#pullrequestreview-1420042464>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ6AOJC3UMT5TZP3IQAO6XLXFNF4ZANCNFSM6AAAAAAX326U7M>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
